frontend/packages/component-library/package.json

```diff
@@ -414,11 +513,13 @@
     "@code-dot-org/fonts": "workspace:*",
     "@code-dot-org/lint-config": "workspace:*",
     "@release-it/conventional-changelog": "^10.0.0",
+    "@storybook/react-webpack5": "^9.0.18",
```

Comment:
Pretty sure these 3 do not need to be added here as they are in the `design-system-storybook`?